### DISCUSSION TITLE
Création du système

### DISCUSSION THREAD

**Discussion by PhilippeLRETQC** [MTESSDev/FRW#11] [Annonces]
Bonjour,

À qui envoyer la demande de création d'un système dans FRW?

Quelques questions complémentaires :

- Je comprend qu'il faut le fichier  default.v0.transmission.yml au minimum. Comme il sera graduable par après avec l'extension d'Azure DevOps et les pipelines. Je suppose qu'il n'est pas grave que l'URL ne soit pas finale?
- Quel est le nombre de caractère au code de système?
- Comment se fera la communication de l'API Key de manière sécuritaire?

Merci beaucoup et bonne journée.

**Comment by j0ber** [comment]
Chez nous c'est @loima02 qui s'occupe des arrimages.
Pouvez-vous nous indiquer qui contacter chez vous s.v.p. ?  Il entrera en contact avec cette personne.


Concernant l'url, en effet vous pourrez la modifier à votre guise, donc ce n'est pas grave qu'il ne sagisse pas de l'url finale.
Les autres éléments seront discutés avec @loima02. Comme il s'agit du 1er arrimage avec un système externe, nous allons en profiter pour documenter le tout au fur et à mesure et rendre le tout accessible dans notre P700.

**Reply by PhilippeLRETQC** [reply]
Bonjour @j0ber et @loima02,

C'est moi qu'il faut contacter (Philippe L.). Merci beaucoup.
